topi/include/topi/nn/upsampling.h

```diff
@@ -0,0 +1,46 @@
+/*!
+ *  Copyright (c) 2017 by Contributors
+ * \file topi/nn/upsampling.h
+ * \brief upsampling op constructors
+ */
+#ifndef TOPI_NN_UPSAMPLING_H_
+#define TOPI_NN_UPSAMPLING_H_
+
+#include <string>
+#include <vector>
+#include <iterator>
+#include <algorithm>
+
+#include "topi/scale.h"
+
+namespace topi {
+namespace nn {
+using namespace tvm;
+
+/*!
+* \brief Upsample given tensor to given shape
+*
+* \param inputs The input tensor array.
+* Bilinear will have 2 inputs one being the weights.
```

Comment:
why do users have to provide bilinear weights? It is not specified in upsampling.cc under NNVM_REGISTER_OP(upsampling) .

can you compute this weight on the fly, given an input tensor?